src/routes/choice-exercises/components/SingleChoice.tsx

```diff
@@ -4,11 +4,11 @@ import styled from 'styled-components/native'
 import { Answer } from '../../../constants/data'
 
 export const StyledContainer = styled.View`
-  padding-top: 6%;
-  height: 42%;
-  margin-horizontal: 8%;
+padding-top: 6%;
+height: 42%;
+margin-left: 8%;
+margin-right: 8%;
```

Comment:
Indentation

Reply:
Can you explain what you mean by Indentation, if you don't mind ?

Reply:
Indentation means, having the correct amount of spaces or tabs. Often running prettier fixes this for you, sometimes you have to fix it manually.